main/src/ca/uwaterloo/flix/language/phase/jvm/GenRefClasses.scala

```diff
@@ -8,7 +8,7 @@ object GenRefClasses {
 
   val ValueFieldName: String = "value"
 
-  def gen(types: List[BackendObjType.Ref])(implicit root: Root, flix: Flix): Map[JvmName, JvmClass] = {
+  def gen(types: Iterable[BackendObjType.Ref])(implicit root: Root, flix: Flix): Map[JvmName, JvmClass] = {
```

Comment:
We never use iterable. Do you mean List?

Reply:
It's actually given a set, but the gen function doesn't really care so I though Iterable was fitting. Should I change it to Set?